manifests/filebeat_oss.pp

```diff
@@ -40,20 +40,11 @@
     require => Package['filebeat'],
   }
 
-  # work around:
-  #  Use cmp to compare the content of local and remote file. When they differ than rm the file to get it recreated by the file resource.
-  #  Needed since GitHub can only ETAG and result in changes of the mtime everytime.
-  # TODO: Include file into the wazuh/wazuh-puppet project or use file { checksum => '..' } for this instead of the exec construct.
-  exec { 'cleanup /etc/filebeat/wazuh-template.json':
-    command => '/bin/rm /etc/filebeat/wazuh-template.json',
-    unless  => "/bin/cmp -s '/etc/filebeat/wazuh-template.json' <(curl -s https://raw.githubusercontent.com/wazuh/wazuh/${wazuh_extensions_version}/extensions/elasticsearch/7.x/wazuh-template.json)",
-  }
-  -> file { '/etc/filebeat/wazuh-template.json':
+  file { '/etc/filebeat/wazuh-template.json':
     owner   => 'root',
     group   => 'root',
-    mode    => '0440',
-    replace => false,  # only copy content when file not exist
-    source  => "https://raw.githubusercontent.com/wazuh/wazuh/${wazuh_extensions_version}/extensions/elasticsearch/7.x/wazuh-template.json",
+    mode    => '0444',
+    content => file('wazuh/wazuh_template_4.3.json'),
```

Comment:
Would not use content with the file function here. The classic way is to use source for this since it is a simple file inside the module.
```suggestion
    source  => "puppet:///modules/${module_name}/wazuh_template_4.3.json",
```

Reply:
Done